pepper-apis/src/main/java/org/broadinstitute/ddp/model/activity/types/EventActionType.java

```diff
@@ -11,7 +11,8 @@
     PDF_GENERATION(false),
     REVOKE_PROXIES(true),
     UPDATE_USER_STATUS(false),
-    USER_ENROLLED(true);
+    USER_ENROLLED(true),
+    UPDATE_CUSTOM_WORKFLOW(false);
```

Comment:
Looks like enum is sorted alphabetically? Can we move the new enum variant up?

Reply:
Fixed